sdk/identity/azure-identity/inc/azure/identity/client_certificate_credential.hpp

```diff
@@ -25,8 +25,7 @@ namespace Azure { namespace Identity {
 
     void FreePkeyImpl(void* pkey);
 
-    template <typename> struct UniquePkeyHelper;
-    template <> struct UniquePkeyHelper<void*>
+    struct UniquePkeyHelper
```

Comment:
Is such a change non-breaking?

Reply:
This one specific is not - it is in Identity's `_detail` namespace.
The broader one, in `unique_handle.hpp` is breaking. It is in `_internal` namespace. If we go with this change, the next Core will break the Identity Beta that we release tomorrow, but it is Beta (both are Betas actually) so it is ok to break.

Reply:
Missed the `_detail` namespace. Makes sense.